spec/support/integration_helpers.rb

```diff
@@ -10,14 +10,15 @@ module IntegrationHelpers
     "bundle exec ruby"
   end
 
-  RSPEC_STUB = File.join(__dir__, "../../bin/rspec")
+  ROOT_DIR = File.expand_path(".")
+  RSPEC_STUB = File.join(ROOT_DIR, "bin/rspec")
 
   def run_rspec(path, chdir: nil, success: true, env: {}, options: "")
-    command = "#{RUBY_RUNNER} #{RSPEC_STUB} #{options} #{path}_fixture.rb"
+    command = "#{RUBY_RUNNER} #{RSPEC_STUB} #{options} spec/fixtures/rspec/#{path}_fixture.rb"
     output, err, status = Open3.capture3(
       env,
       command,
-      chdir: chdir || File.expand_path("../../fixtures/rspec", __FILE__)
```

Comment:
What's the purpose of these changes?

Reply:
I made these changes because I wanted to run the `command` right from the terminal and to make the `command` variable look "prettier".


This is how it used to be:
```
bundle exec ruby /home/yu/projects/mock-suey/spec/support/../../bin/rspec  double_fixture.rb
```
which runs from `spec/fixtures/rspec`. There's a part `../../`, which I removed.


This is how it looks now:
```
bundle exec ruby /home/yu/projects/mock-suey/bin/rspec  spec/fixtures/rspec/stored_mocked_calls_fixture.rb
```
It runs from project root.